shared/torngit/base.py

```diff
@@ -219,8 +219,8 @@ def diff_to_json(self, diff):
                 elif sol4 == "@@ -":
                     # ex: "@@ -31,8 +31,8 @@ blah blah blah"
                     # ex: "@@ -0,0 +1 @@"
-                    l = get_start_of_line(source).groups()
```

Comment:
"l" is an ambiguous character, and rightfully so. changed to "ln" to signal "line" which is what "l" was originally representing